{{ cookiecutter.repo_name }}/tox.ini

```diff
@@ -20,7 +20,7 @@ skip_install = true
 commands =
     bash -c 'poetry export --with dev --without-hashes -f requirements.txt | grep -v "^[dD]jango==" | grep -v "^psycopg2-binary==" | pip install --no-compile -q --no-deps -r /dev/stdin'
     pip install --no-compile -q --no-deps --no-build-isolation -e .
-    pytest --create-db --cov --cov-fail-under=0 --cov-append --cov-config pyproject.toml {{ cookiecutter.module_name }}/
```

Comment:
The old limitation to the modulepath isn’t necessary due to `testpaths` being configured: https://github.com/Opus10/public-django-app-template/blob/61fb54bab32508ca0cc5d70a9155f49d124f04c0/%7B%7B%20cookiecutter.repo_name%20%7D%7D/pyproject.toml#L84